.github/workflows/docker-build-test.yaml

```diff
@@ -25,16 +25,6 @@ on: # build on main branch OR when a PR is labeled with `CICD:build-images`
   workflow_dispatch:
   pull_request_target:
     types: [labeled, opened, synchronize, reopened, auto_merge_enabled]
-  # For PR that modify .github, run from that PR
```

Comment:
Why doesn't this hold anymore @perryjrandall ?

Reply:
This was triggering duplicate forge runs on PR that would contend and cause the job to fail to merge

We need a better way to test ci and in general a better ci strategy here